index.html

```diff
@@ -52,7 +52,113 @@ <h1>  I’m Lalla <br>
               </ul>
           
           </section>
-         
+          <div class="grid-container">
+            <section class = "works">
+                <div class="job">
+                    <img src="image/Snapshoot-Portfolio.jpg" > </div>
+                    <div class="content">
+                        <h3>Tonic</h3>
+                        <ul>
+                            <li>Canopy</li>
+                            <li>Back End DEv</li>
+                            <li>2015</li>
+                        </ul>
+                        <p class="Descriptions">A daily selection of privately <br> personalized reads; no accounts or sign-ups required.</p>
+                        <ul>
+                            <li><a href="#" class="programming">html</a></li>
+                            <li><a href="#" class="programming">css</a></li>
+                            <li><a href="#" class="programming">javascript</a></li>
+                        </ul>
+                        
+                        <button type="button">See Project</button>
+                   
+                </div>
+                <div class="job">
+                  <img src="image/Snapshoot-Portfolio-2.jpg" alt="job-2 image">
+                  <div class="content">
```

Comment:
- [ ] According to the project's requirements you need to use `display: flex` to lay out the content of your cards, please make sure to do that.